html/verse_context_menu.html

```diff
@@ -0,0 +1,18 @@
+<div class="assign-last-tag-button fg-button ui-state-default ui-corner-all ui-state-disabled">
+  <span i18n="tags-toolbar.assign-last-tag"></span>
+</div>
+
+<div class="edit-note-button fg-button ui-state-default ui-corner-all ui-state-disabled">
+  <i class="fas fa-comment-alt"></i>
+  <span i18n="bible-browser.edit-note"></span>
+</div>
+
+<div class="copy-clipboard-button fg-button ui-state-default ui-corner-all ui-state-disabled">
+  <i class="fas fa-clipboard"></i>
+  <span i18n="bible-browser.copy"></span>
+</div>
+
+<div class="show-context-button fg-button ui-state-default ui-corner-all ui-state-disabled">
+  <i class="fas fa-arrows-alt-v"></i><i class="fas fa-align-justify"></i>
+  <span i18n="tags-toolbar.context"></span>
+</div>
```

Comment:
This can be a web-component. 
It would be much easier to extend/refractor it if all view-layout, css style and js logic would be in one place (file).